examples/example_write_serial_number.py

```diff
@@ -0,0 +1,37 @@
+"""Example for writing an individual address to a KNX device."""
+import asyncio
+import sys
+
+from xknx import XKNX
+from xknx.management import procedures
+from xknx.telegram import IndividualAddress
+
+
+async def main(argv: list[str]) -> int:
+    """
+    Write the individual address to a device in programming mode.
+
+    This fails if multiple devices are in programming mode and/or when there is no device found in programming mode.
```

Comment:
Update comment

Reply:
Fixed in d5e2a7247180e5531be50748e29f6b1c45a93b2d